Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -615,6 +616,10 @@ class RenderStyle {
     bool isGridAutoFlowDirectionColumn() const { return (m_rareNonInheritedData->grid->gridAutoFlow & InternalAutoFlowDirectionColumn); }
     bool isGridAutoFlowAlgorithmSparse() const { return (m_rareNonInheritedData->grid->gridAutoFlow & InternalAutoFlowAlgorithmSparse); }
     bool isGridAutoFlowAlgorithmDense() const { return (m_rareNonInheritedData->grid->gridAutoFlow & InternalAutoFlowAlgorithmDense); }
+    bool isMasonryAutoFlowPack() const { return (m_rareNonInheritedData->grid->masonryAutoFlow.contains(MasonryAutoFlow::Pack)); }
+    bool isMasonryAutoFlowNext() const { return (m_rareNonInheritedData->grid->masonryAutoFlow.contains(MasonryAutoFlow::Next)); }
+    bool isMasonryAutoFlowDefiniteFirst() const { return (m_rareNonInheritedData->grid->masonryAutoFlow.contains(MasonryAutoFlow::DefiniteFirst)); }
+    bool isMasonryAutoFlowOrdered() const { return (m_rareNonInheritedData->grid->masonryAutoFlow.contains(MasonryAutoFlow::Ordered)); }
```

Comment:
Not sure if these individual accessors are necessary. How about just exposing masonryAutoFlow() OptioSet? Doing `masonryAutoFlow().contains(MasonryAutoFlow::Pack)` etc in call sites would not be overly verbose.

Reply:
Removed